packages/backend-for-frontend/src/api/eserviceTemplateApiConverter.ts

```diff
@@ -106,3 +108,14 @@ export const toBffCreatedEServiceTemplateVersion = (
     versionId: version.id,
   };
 };
+
+export function toCatalogCreateEServiceTemplateSeed(
+  eServiceTemplateSeed: bffApi.EServiceTemplateSeed
+): eserviceTemplateApi.EServiceTemplateSeed {
+  return {
+    ...eServiceTemplateSeed,
+    version: {
+      voucherLifespan: 60,
```

Comment:
The same happens for the e-service creation (/interop-be-monorepo/packages/backend-for-frontend/src/api/catalogApiConverter.ts) a default value of 60 is given for the voucherLifespan